gradle.properties

```diff
@@ -0,0 +1,2 @@
+grailsVersion=4.0.13
```

Comment:
Please update to be 4.1.1  since < 4.1.1 has a security vulnerability

Reply:
updated to 4.1.1